pkg/asset/installconfig/aws/regions.go

```diff
@@ -1,32 +1,43 @@
 package aws
 
 import (
-	"github.com/aws/aws-sdk-go/aws/endpoints"
+	"context"
 
-	"github.com/openshift/installer/pkg/rhcos"
-	"github.com/openshift/installer/pkg/types"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/ec2"
+	"github.com/sirupsen/logrus"
+	"k8s.io/apimachinery/pkg/util/sets"
 )
 
 // knownPublicRegions is the subset of public AWS regions where RHEL CoreOS images are published.
 // This subset does not include supported regions which are found in other partitions, such as us-gov-east-1.
-// Returns: a map of region identifier to region description.
-func knownPublicRegions(architecture types.Architecture) map[string]string {
-	required := rhcos.AMIRegions(architecture)
```

Comment:
I don't think we want to remove the rhcos stream as a source of truth for AWS regions in the survey. This basically suggests a certain level of support for a region in openshift, and it is why we use it as the standard for what to display in the survey.